.github/workflows/kind-k8s-versions-check.yml

```diff
@@ -1,54 +0,0 @@
-# Checks the latest kind image version from `https://registry.hub.docker.com/v2/repositories/kindest/node/tags`,
```

Comment:
not sure why we're deleting this workflow. There's a public cloud versions check in CNP that wer're not bringing in instead. Wondering if we forgot to.